packages/rules-unit-testing/package.json

```
@@ -15,7 +15,7 @@
     "build:deps": "lerna run --scope @firebase/rules-unit-testing --include-dependencies build",
     "dev": "rollup -c -w",
     "test:nyc": "TS_NODE_CACHE=NO TS_NODE_COMPILER_OPTIONS='{\"module\":\"commonjs\"}' nyc --reporter lcovonly -- mocha 'test/{,!(browser)/**/}*.test.ts' --config ../../config/mocharc.node.js",
-    "test": "firebase --project=foo --debug emulators:exec 'yarn test:nyc'",
+    "test": "FIREBASE_CLI_PREVIEWS=storageemulator STORAGE_EMULATOR_HOST=http://localhost:9199 firebase --project=foo --debug emulators:exec 'yarn test:nyc'",
```

I had to add the `STORAGE_EMULATOR_HOST` variable to compensate for https://github.com/firebase/firebase-tools/pull/3337

---

```suggestion
    "test": "FIREBASE_CLI_PREVIEWS=storageemulator firebase --project=foo --debug emulators:exec 'yarn test:nyc'",
```

Gotta clean this up now we've released the CLI with the fix